internal/infrastructure/kubernetes/deployment.go

```diff
@@ -35,24 +36,20 @@ const (
 	envoyHTTPPort = int32(8080)
 	// envoyHTTPSPort is the container port number of Envoy's HTTPS endpoint.
 	envoyHTTPSPort = int32(8443)
+	// envoyGatewayXdsServerHost is the DNS name of the Xds Server within Envoy Gateway.
+	// It defaults to the Envoy Gateway Kubernetes service.
+	envoyGatewayXdsServerHost = "envoy-gateway"
+	// envoyAdminAddress is the listening address of the envoy admin interface.
+	envoyAdminAddress = "127.0.0.1"
```

Comment:
should this be configurable, right now it's hard to debug via `config_dump`

Reply:
@zirain we discussed making the bootstrap configurable post v0.2.0. Let me add it to the agenda for the community meeting.